pom.xml

```diff
@@ -477,6 +498,14 @@
 					</execution>
 				</executions>
 			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-compiler-plugin</artifactId>
+				<configuration>
+					<source>11</source>
```

Comment:
Does one of the new libraries require JDK 11? The upstream app requires JDK 8 so I'd rather stick with that if we can